.vale/styles/OpenShiftAsciiDoc/SuggestAttribute.yml

```diff
@@ -25,13 +25,23 @@ swap:
   cert-manager Operator for Red Hat OpenShift: "{cert-manager-operator}"
   Cluster API Provider AWS: "{cap-aws-first}|{cap-aws-short}"
   Cluster API Provider Azure: "{cap-azure-first}|{cap-azure-short}"
+  Cluster API Provider for AWS: "{cap-aws-first}|{cap-aws-short}"
+  Cluster API Provider for Azure: "{cap-azure-first}|{cap-azure-short}"
+  Cluster API Provider for GCP: "{cap-gcp-first}|{cap-gcp-short}"
+  Cluster API Provider for IBM Cloud: "{cap-ibm-first}|{cap-ibm-short}"
+  Cluster API Provider for Kubevirt: "{cap-kubevirt-first}|{cap-kubevirt-short}"
+  Cluster API Provider for Nutanix: "{cap-nutanix-first}|{cap-nutanix-short}"
+  Cluster API Provider for OCI: "{cap-oci-first}|{cap-oci-short}"
+  Cluster API Provider for OpenStack: "{cap-openstack-first}|{cap-openstack-short}"
+  Cluster API Provider for vSphere: "{cap-vsphere-first}|{cap-vsphere-short}"
   Cluster API Provider GCP: "{cap-gcp-first}|{cap-gcp-short}"
   Cluster API Provider IBM Cloud: "{cap-ibm-first}|{cap-ibm-short}"
   Cluster API Provider Kubevirt: "{cap-kubevirt-first}|{cap-kubevirt-short}"
   Cluster API Provider Nutanix: "{cap-nutanix-first}|{cap-nutanix-short}"
   Cluster API Provider OCI: "{cap-oci-first}|{cap-oci-short}"
   Cluster API Provider OpenStack: "{cap-openstack-first}|{cap-openstack-short}"
   Cluster API Provider vSphere: "{cap-vsphere-first}|{cap-vsphere-short}"
+  Cluster CAPI Operator: "{cluster-capi-operator}"
```

Comment:
Cross-checked these attributes with the common-attributes file and didn't see an attribute for this one. I presume you have this in hand though :-)

Reply:
lgtm apart from that, I can merge when you're ready

Reply:
I'm having one of those days...